ReactNativeClient/lib/models/BaseItem.js

```diff
@@ -644,7 +644,7 @@ class BaseItem extends BaseModel {
 
 	static displayTitle(item) {
 		if (!item) return '';
-		return !!item.encryption_applied ? '🔑 ' + _('Encrypted') : item.title + '';
+		return !!item.encryption_applied ? '🔑 ' + _('Encrypted') : (!!item.title)? item.title + '' : Note.defaultTitle(item);
```

Comment:
Turns out this broke the apps since `Note` is not imported in this class, nor should it be since this is the base class. I'll implement a fix for it.

Reply:
Sorry about this, I'll have to be more careful.

Reply:
No problem, it can happen to anybody! And I should have spotted the issue earlier. For now, I've simply changed it to display "Untitled" when the title is undefined.